make_wheels.py

```diff
@@ -134,20 +134,33 @@ def write_ziglang_wheel(out_dir, *, version, platform, archive):
         name='ziglang',
         version=version,
         tag=f'py3-none-{platform}',
-        metadata={
-            'Summary': 'Zig is a general-purpose programming language and toolchain for maintaining robust, optimal, and reusable software.',
-            'Description-Content-Type': 'text/markdown',
-            'License': 'MIT',
-            'Classifier': [
-                'License :: OSI Approved :: MIT License',
-            ],
-            'Project-URL': [
-                'Homepage, https://ziglang.org',
-                'Source Code, https://github.com/ziglang/zig-pypi',
-                'Bug Tracker, https://github.com/ziglang/zig-pypi/issues',
-            ],
-            'Requires-Python': '~=3.5',
-        },
+        metadata=[
+            ('Summary', 'Zig is a general-purpose programming language and toolchain for maintaining robust, optimal, and reusable software.'),
+            ('Description-Content-Type', "'text/markdown'; charset=UTF-8; variant=GFM"),
+            ('License-Expression', 'MIT'),
+            ('License-File', 'LICENSE'),
+            ('License-File', 'ziglang/lib/libc/glibc/LICENSES'),
+            ('License-File', 'ziglang/lib/libc/mingw/COPYING'),
+            ('License-File', 'ziglang/lib/libc/musl/COPYRIGHT'),
+            ('License-File', 'ziglang/lib/libc/wasi/LICENSE'),
+            ('License-File', 'ziglang/lib/libc/wasi/LICENSE-APACHE'),
+            ('License-File', 'ziglang/lib/libc/wasi/LICENSE-APACHE-LLVM'),
+            ('License-File', 'ziglang/lib/libc/wasi/LICENSE-MIT'),
+            ('License-File', 'ziglang/lib/libcxx/LICENSE.TXT'),
+            ('License-File', 'ziglang/lib/libcxxabi/LICENSE.TXT'),
+            ('License-File', 'ziglang/lib/libunwind/LICENSE.TXT'),
+            ('Classifier', 'Development Status :: 4 - Beta'),
+            ('Classifier', 'Intended Audience :: Developers'),
+            ('Classifier', 'Topic :: Software Development :: Compilers'),
+            ('Classifier', 'Topic :: Software Development :: Code Generators'),
+            ('Classifier', 'Topic :: Software Development :: Build Tools'),
+            ('Classifier', 'Programming Language :: Other'),
+            ('Classifier', 'Programming Language :: Other Scripting Engines'),
+            ('Project-URL', 'Homepage, https://ziglang.org'),
+            ('Project-URL', 'Source Code, https://github.com/ziglang/zig-pypi'),
+            ('Project-URL', 'Bug Tracker, https://github.com/ziglang/zig-pypi/issues'),
+            ('Requires-Python', '~=3.5'),
```

Comment:
https://endoflife.date/python states that security releases for Python 3.7 have ended. Now that 3.13 is just around the corner, maybe we should update to a minimum of 3.8 (either here or as a separate PR). Please let me know.

Reply:
I'm aware of the EOL status of Python 3.5 (it was EOL when I wrote this package). I decided that this requirement should be bumped only if the package isn't installable/usable, since there isn't really a cost to keeping it low, and it could help someone using old Python versions.

Reply:
Sure, makes sense!

Reply:
If we do change the launcher in such a way it stops working on earlier Python versions, we should support the last non-EOL version, of course.

https://github.com/ziglang/zig-pypi/blob/f1cd7a919394f7b103ab9f6ecc9100ed9e37e173/make_wheels.py#L122-L127

Reply:
I can't access a Python version that old unless I use `actions/setup-python` in a CI setup – and I remember that your previous mention in our earlier conversations of one to not become a maintenance burden. So we could leave it until someone asks; I doubt anyone will, though, since those Pythons are _really_ old 😄